addons/main/cfg3DEN.hpp

```diff
@@ -107,25 +110,6 @@ class Cfg3DEN
                 };
             };
         };
-        class EditCodeMulti5: EditCodeMulti3
-        {
-            h = QUOTE((5 + 20 * 3.5) * GRID_H);
-            class Controls: Controls
-            {
-                class Background: Background
-                {
-                    h = QUOTE((20 * 3.5 + 0.6 * 5) * GRID_H);
-                };
-                class Value: Value
-                {
-                    h = QUOTE(20 * 3.5 * GRID_H - 1 * GRID_H);
-                };
-                class Title: Title
-                {
-                    h = QUOTE((20 * 3.5 + 1 * 5) * GRID_H);
-                };
-            };
-        };
```

Comment:
See https://github.com/R3voA3/3den-Enhanced/pull/337/commits/55605921bd2cb5876f390c28706c932870d0a130 for details. TL;DR, these same changes are inherited from EditMulti5.